pages/k8s/auth.md

```diff
@@ -0,0 +1,168 @@
+root     18966 18964  0 12:11 ?        00:00:00 grep apiserver
+```
+
+## Setting a configuration
+
+The authorisation mode can be set using the same **Juju** command as above, but this
+time specifying a value:
+
+```bash
+juju config kubernetes-master authorization-mode="Node"
+```
+
+It is possible to set more than one mode using a comma-separated list:
+
+```bash
+juju config kubernetes-master authorization-mode="RBAC, Node"
+```
+
+<div class="p-notification--positive"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Note:</span>
+Using "RBAC,Node" for authorisation is the recommended configuration.
+</p></div>
+
+The order matters. Kubernetes will process each API request with each module in
+sequence. If the current authorising module either allows or denies the
+request, that decision is made and no further modules are consulted. If the
+current module has no opinion on the request, then the decision is passed to
+the next module in the list. If no decision has been returned by the last
+module in the list, then the request is denied.
+
+
+
+<a id='rbac'> </a>
+
+## Further information on RBAC
+
+Many of the defined roles (those prefixed by 'system:' ) for RBAC are really intended for
+managing access to Kubernetes componenets themselves.
+
+The main  cluster roles for additional users are:`admin`, `cluster-admin`, `edit` and `view` .
+
+You can view the available roles,  cluster roles and bindings with the following commands:
+
+```bash
+kubectl get roles --all-namespaces
+kubectl get clusterroles --all-namespaces
+kubectl get rolebindings --all-namespaces
+kubectl get clusterrolebindings  --all-namespaces
+```
+
+### Adding or removing users and editing roles
+
+The recommended method for managing Kubernetes users is through an external
+authentication service such as LDAP (see the documentation on
+[using LDAP with CDK][docs-ldap]). While Kubernetes supports such authentication,
```

Comment:
I would leave out this sentence: "While Kubernetes supports such authentication, the implementation is left up to the user."

Reply:
:+1: